homeassistant/components/onvif/manifest.json

```diff
@@ -3,12 +3,11 @@
   "name": "Onvif",
   "documentation": "https://www.home-assistant.io/components/onvif",
   "requirements": [
-    "onvif-py3==0.1.3",
-    "suds-passworddigest-homeassistant==0.1.2a0.dev0",
-    "suds-py3==1.3.3.0"
+    "onvif-zeep-async==0.1.1",
+    "zeep[async]==3.0.0"
```

Comment:
Why is `zeep[async]` a requirement? Do you use it explicitly ? It's also a dependency of onvif-zeep-async.